learning/workflow/assets/importing_scenes.rst

```diff
@@ -255,7 +255,7 @@ Remove nodes (-noimp)
 Node names that have this suffix will be removed at import time, mo
 matter what their type is. They will not appear in the imported scene.
 
-Create collisions (-col, -colonly)
+Create collisions (-col, -colonly, -convcolonly)
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
You also need to adjust the length of the underline, otherwise Sphinx throws warnings.